botocore/data/autoscaling/2011-01-01/waiters-2.json

```diff
@@ -0,0 +1,62 @@
+{
+    "version": 2,
+    "waiters": {
+        "GroupExists": {
+            "acceptors": [
+                {
+                    "argument": "length(AutoScalingGroups) > `0`",
```

Comment:
Bringing in relevant parties: @awood45 @jeskew

@jamesls do you mean that it will omit it if it does not exist thus possibly returning a partial result? If that is the case, would it be impossible to wait for multiple groups because we cannot inject input parameters to the acceptors using our current waiters spec since we would need to know how many groups the user specified?